book/website/community-handbook/style/style-videos.md

```diff
@@ -6,22 +6,14 @@
 If your video is hosted on a platform like YouTube or Vimeo you can use the `iframe` directive to embed it.
 For example,
 
-````
-```{iframe} https://www.youtube.com/embed/MdOS6tPq8fc
+::: {iframe} https://www.youtube.com/embed/MdOS6tPq8fc
```

Comment:
This one is a demo - the ```` mean that it renders as code rather than as a video, so these should be left in

Reply:
Hi @tongyu0924 - for this one, the content inside the ```` should match the new way that we're recommend doing the video embeds (using the ::: syntax rather than the previous ``` syntax)